src/openInBrowser.js

```diff
@@ -0,0 +1,14 @@
+const opn = require('opn');
+
+const openInBrowser = (url, browser) => {
+  try {
+    const options = typeof browser === 'string' ? {app: browser} : undefined;
+
+    opn(url, options).catch(() => {}); // Prevent `unhandledRejection` error.
+    return true;
+  } catch (err) {
+    return false;
```

Comment:
Don’t you think it would be better if we told the user about the error? We probably can’t use the Logger class since it hasn't been initialized yet, but we can at least do something simple like:

```javascript
console.error('Unexpected error when opening the browser:', err)
```

That way we don't throw (which would prevent bundling), but we still inform the user about the error.

Reply:
sure sound good.